packages/core/src/external/fhir/patient/index.ts

```diff
@@ -78,14 +78,22 @@ export function mapPatientDataToResource(patient: PatientIdAndData) {
     gender: mapMetriportGenderToFhirGender(patient.data.genderAtBirth),
     birthDate: patient.data.dob,
     address:
-      patient.data.address.map((addr: Address) => {
-        return {
-          line: addr.addressLine1 ? [addr.addressLine1] : [],
+      patient.data.address.flatMap((addr: Address) => {
+        const baseAddress = {
           city: addr.city,
           state: addr.state,
           postalCode: addr.zip,
           country: addr.country || "USA",
         };
+
+        if (addr.addressLine2) {
+          return [
+            { ...baseAddress, line: [addr.addressLine1] },
+            { ...baseAddress, line: [addr.addressLine1, addr.addressLine2].filter(Boolean) },
```

Comment:
if addressLine2 is defined, lets create a second address with both of them. The reason is that there is no penalty in epic for adding more addresses, and we arent sure the exact way to join addressLine1 and addressLine2. This approach is net positive wiuth no negative consequences basically